test/test_dictionaries.py

```diff
@@ -0,0 +1,28 @@
+import sys
+
+sys.path.append('../cjs_capstone')
+
+import unittest2
+import main
+
+class TestDictionaries(unittest2.TestCase):
```

Comment:
Really like that you added the separate test file and what you have done with tests here!